flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
Unity Linux builds only supports x86_64 :/

Reply:
Yeah, that's annoying.